### DISCUSSION TITLE
next build command throws 'child_process' error and stops execution

### DISCUSSION THREAD

**Discussion by harshdoshi999** [vercel/next.js#15410] [Help]
# Bug report
next build command throws 'child_process' error and stops execution

## Describe the bug

I'm using next v9.4.4
When I try to run next build, it throws child_process error and stops execution.

Exact error:
Module not found: Can't resolve 'child_process' in '\node_modules\google-auth-library\build\src\auth'.
If I remove google-auth-library module then same error shows for node-mailer module

## System information

- OS: [Windows 10]
- Browser (if applies) [Chrome]
- Version of Next.js: [9.4.4]
- Version of Node.js: [10.15.4]

## Additional context

However nodemon server/app.js works fine but next build does not work

**Comment by jamesmosier** [comment]
Hi @harshdoshi999. It sounds like those packages are meant to be run in a Node environment. Could you create a repository that recreates this issue?
